examples/text-generation/run_generation.py

```diff
@@ -267,7 +267,7 @@ def setup_parser(parser):
         "--ignore_eos",
         default=True,
         action=argparse.BooleanOptionalAction,
-        help="Whether to ignore eos, set False to disable it",
+        help="Whether to disable stopping with eos token when calling `generate`. --no-ignore_eos to disable it",
```

Comment:
**Minor**: For `argparse.BooleanOptionalAction`, if we have `--foo`, then the arg parser automatically provides `--no-foo`. That's mentioned in argparse documentation: https://docs.python.org/3/library/argparse.html ,  so we might not need to explicitly add `--no-ignore_eos to disable it` here.

I think its fine and good to add it, so we can keep it. But maybe we should add it in the help line of `examples/image-to-text/run_pipeline.py` as well. Or we should remove mention of `--no-ignore_eos` from both `examples/text-generation/run_generation.py` and `examples/image-to-text/run_pipeline.py`

Reply:
The `ignore_eos` arg in `examples/text-generation/run_generation.py` does not use `argparse.BooleanOptionalAction`, it's `action="store_true",`, default value is false. and adding `--ignore_eos` in command makes it true. so no need to mention --no-ignore_eos in `examples/text-generation/run_generation.py`